package.json

```diff
@@ -39,12 +39,14 @@
     "electron-dl": "^1.6.0",
     "formik": "^0.10.4",
     "from2": "^2.3.0",
+    "i": "^0.3.6",
     "install": "^0.10.2",
     "jayson": "^2.0.2",
     "jshashes": "^1.0.7",
     "keytar-prebuild": "^4.0.4",
+    "lbry-redux": "lbryio/lbry-redux#common-components-refactor",
     "localforage": "^1.5.0",
-    "npm": "^5.5.1",
+    "npm": "^5.6.0",
```

Comment:
We're using `yarn` instead of `npm` for the project. We should probably get rid of `npm` as a dependency.